charon/pkgs/radas_signature_handler.py

```diff
@@ -33,14 +33,23 @@ class RadasReceiver(MessagingHandler):
     This receiver will listen to UMB message queue to receive signing message for
     signing result.
     Attributes:
-        sign_result_loc (str): Local save path (e.g. “/tmp/sign”) for oras pull result,
-        this value transfers from the cmd flag, should register UmbListener when the client starts
+        sign_result_loc (str):
+            Local save path (e.g. “/tmp/sign”) for oras pull result, this value transfers
+            from the cmd flag,should register UmbListener when the client starts
+        request_id (str):
+            Identifier of the request for the signing result
+        sign_result_status (str):
+            Result of the signing(success/failed)
+        sign_result_errors (list):
+            Any errors encountered if signing fails, this will be empty list if successful
     """
 
     def __init__(self, sign_result_loc: str, request_id: str) -> None:
         super().__init__()
         self.sign_result_loc = sign_result_loc
         self.request_id = request_id
+        self.sign_result_status = "success"
```

Comment:
This does not make sense, because it is asynchronized accessed in main process. If we set it in init method, it means it will be "success" at start time, even that the receiver does not get any message yet from UMB. Instead we should set it as None, then loop and waiting to check it to be a not None value to continue.